src/main/java/Main.java

```diff
@@ -21,20 +21,22 @@ public static void main(String[] args) throws IllegalAccessException {
 //        TestClass.class.getDeclaredField("b").setAccessible(true);
 //        TestClass.class.getDeclaredField("b").set(testClass, 1);
         System.out.println(testClass);
+
 //        for (var i : Arrays.stream(TestClass.class.getDeclaredFields()).map(Field::getGenericType).toList()){
 //            testClass.
 //        }
         for (var method : Arrays.stream(TestClass.class.getDeclaredMethods()).filter(method -> method.getName().startsWith("set")).toList()) {
             try {
                 Class<?> type = method.getParameterTypes()[0];
-                System.out.println(type.getName());
```

Comment:
let's use logger instead of system.out

Reply:
this is just the playground, wasnt gonna commit it at all!! but fine